src/eve/concepts.py

```diff
@@ -233,53 +245,56 @@ class Config(FrozenModelConfig):
         pass
 
 
-class UIDGenerator:
-    """Simple unique id generator using a counter."""
+AnyNodeClasses = (bool, bytes, int, float, str, IntEnum, StrEnum, Node, list, dict, set)
```

Comment:
Shouldn't this be placed along the other type definitions above?

Reply:
This is a leftover from a previous version. It will be deleted.